charts/gardener-extension-networking-cilium/values.yaml

```diff
@@ -47,6 +47,11 @@ config:
 
 gardener:
   version: ""
+  seed:
+    spec:
+      settings:
+        topologyAwareRouting:
+          enabled: false
```

Comment:
The release note suggests that the hints will be enabled depending on the Seed setting and the Shoot HA failure tolerance type. Then, I would expect this value to be adapted programmatically during the reconciliation flow, but it looks to be statically `false`.

Could you please share how this should work if this is always `false`?